src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs

```diff
@@ -459,6 +459,13 @@ private BoundExpression BindArgListOperator(InvocationExpressionSyntax node, Bin
                                     methodGroup.ResultKind);
                             break;
                     }
+
+                    if (receiver.Type.IsRefLikeType)
```

Comment:
Is it just receivers though? Won't the same issue popup if any argument is a `ref struct`?

Reply:
We already block that: https://github.com/dotnet/roslyn/blob/main/src/Compilers/CSharp/Portable/Binder/Binder_Invocation.cs#L590.